src/test/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/entities/payment/PaymentDtoTest.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.payment;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.fee.FeeDto;
+
+class PaymentDtoTest {
+
+    private final String id = "id";
+    private final BigDecimal amount = new BigDecimal("140");
+    private final String description = "Appeal with hearing";
+    private final String reference = "RC-1627-5070-9329-7815";
+    private final String currency = "GBP";
+    private final String ccdCaseNumber = "1627506765384547";
+    private final String channel = "online";
+    private final String method = "card";
+    private final String externalProvider = "gov pay";
+    private final String externalReference = "8saf7t8kav53mmubrff738nord";
+    private final String siteId = "BFA1";
+    private final String status = "Success";
+
+    private PaymentDto paymentDto;
+
+    @Test
+    void should_hold_onto_values() {
+
+        paymentDto = PaymentDto.builder()
+            .id(id)
+            .amount(amount)
+            .description(description)
+            .reference(reference)
+            .service(Service.IAC)
+            .dateCreated(new Date())
+            .dateUpdated(new Date())
+            .currency(currency)
+            .ccdCaseNumber(ccdCaseNumber)
+            .channel(channel)
+            .method(method)
+            .externalProvider(externalProvider)
+            .externalReference(externalReference)
+            .siteId(siteId)
```

Comment:
@tarun-palisetty can replace with `case_type`

Reply:
Removed siteId check in the test